office_space_allocation/main.py

```diff
@@ -0,0 +1,300 @@
+import os
+import re
+import sqlite3
+import subprocess
+import sys
+from colorama import Fore, Style
+
+from tabulate import tabulate
+
+from office_space_allocation import amity, office, staff, livingspace, fellow, utilities, db
+
+main_amity = None
```

Comment:
Actually, that makes sense. Since I only need a single instance of Amity class `main_amity = amity.Amity()` will work. Will refactor that. Thanks again.